cmd/tailscale/cli/status.go

```diff
@@ -160,40 +160,55 @@ func runStatus(ctx context.Context, args []string) error {
 		)
 		relay := ps.Relay
 		anyTraffic := ps.TxBytes != 0 || ps.RxBytes != 0
-		var offline string
-		if !ps.Online {
-			offline = "; offline"
-		}
+
+		lastseen := CalculateLastSeenTime(ps)
+
+		// prepare status components to join consistently with "; "
+		var statusParts []string
+
 		if !ps.Active {
 			if ps.ExitNode {
-				f("idle; exit node" + offline)
+				statusParts = append(statusParts, "idle")
+				statusParts = append(statusParts, "exit node")
```

Comment:
Blocking: Aren’t you only changing what is rendered by `offline`? Is it necessary to reimplement as a joined string slice?